boefjes/boefjes/clients/bytes_client.py

```diff
@@ -99,17 +100,25 @@ def get_normalizer_meta(self, normalizer_meta_id: uuid.UUID) -> NormalizerMeta:
 
     @retry_with_login
     def save_raw(self, boefje_meta_id: str, raw: str | bytes, mime_types: Set[str] = frozenset()) -> UUID:
-        headers = {"content-type": "application/octet-stream"}
-        headers.update(self.headers)
+        file_name = "raw"  # The name provides a key for all ids returned, so this is arbitrary as we only upload 1 file
```

Comment:
Theres a ticket somewhere for naming raw files, as naming them seems logical as a feature and useful for the user in some situations where one job has multiple raw files with the same mime-types..

Reply:
Oki, let me know if you can find it and I'll take a look at that one as well